src/runners/express.js

```diff
@@ -38,6 +38,44 @@ function createRunner(req, startTime) {
     return expressEvent;
 }
 
+/**
+ * Check if parametered path and absolute path are equal
+ * @param {Array} parameteredPathSplitted parametered splitted path
+ * @param {Array} pathSplitted absolete splitted path
+ * @return {Boolean} true if equal false if not.
+ */
+function checkIfPathsAreEqual(parameteredPathSplitted, pathSplitted) {
+    for (let i = 0; i < pathSplitted.length; i += 1) {
+        if (parameteredPathSplitted[i] !== pathSplitted[i] &&
+            parameteredPathSplitted[i] &&
+            parameteredPathSplitted[i][0] !== ':') {
+            return false;
+        }
+    }
+
```

Comment:
there can be an index out of bounds exception here (make sure the length is the same first)

Reply:
I check it in findCalledParameteredPath